### DISCUSSION TITLE
Best way to take a snapshot of a SurfaceMesh  ?

### DISCUSSION THREAD

**Discussion by maximecharriere** [CGAL/cgal#6427] [Q&A]
Hello,
I'm using CGAL and I have a SurfaceMesh with colors that I need to make a snapshot in .jpg or .png format.  
I don't think so, but is there a way to do it with CGAL or a compatible library?  

If this is not possible directly from a SurfaceMesh, what would be your advice to achieve this?  

I could also export it in .ply format and make an image with another library or software. Do you know any?

**Comment by afabri** [comment]
In case you have compiled the Polyhedron demo, go to File -> Save Snapshot

**Reply by maximecharriere** [reply]
Thanks for the quick answer!
The only problem is that the snapshot should be taken automatically from the code. I don't have a GUI.
I should be able to do `mesh.saveSnapshot()` .  I will look if I can easily implement saveSnapshot() in my code.  

Do you have any other ideas?
